wandb/integration/weave/weave.py

```diff
@@ -43,6 +49,14 @@ def setup(entity: str | None, project: str | None) -> None:
     # If weave is not yet imported, we can't init it from here.  Instead, we'll
     # rely on the weave library itself to detect a run and init itself.
     if _WEAVE_PACKAGE_NAME not in sys.modules:
+        # Check if any target library is imported and suggest Weave installation
+        imported_libs = [lib for lib in TARGET_LIBS if lib in sys.modules]
+        if imported_libs:
+            wandb.termlog(
+                f"Detected [{', '.join(imported_libs)}] in use.\n"
+                "To get improved LLM call tracking, `pip install weave` then add `import weave` to the top of your script.\n"
```

Comment:
Limit all lines to 80 characters, please. Instead of inserting `\n` characters, consider making each line a call to `wandb.termlog()`.